### PR TITLE
P0 homepage

### PR DESCRIPTION
**Pull request by purvasharmadev**
Added the homepage:

-navbar
-featured-product section
-landing page
-new-products section

kstyle-home-page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    width: 100%;
+    text-align: center;
+    cursor: pointer;
+    opacity: 0.8;
+}
+.flex-container{
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: nowrap;
+    max-width: 1300px;
```
Comment (bossmonkey98) [inline index.css]: Use rem instead of px

Comment (purvasharmadev) [reply index.css]: Thank you for pointing it out. Fixed!

File: index.css [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    position: fixed;
+    top: 0;
+    z-index:1;
+}
+header{
+    background-image:linear-gradient(45deg,var(--SECONDARY-COLOR),rgba(0, 0, 0, 0.178) ), url("https://variety.com/wp-content/uploads/2021/05/BTS_Butter-photo.jpg");
+    height: 100vh;
+    background-repeat: no-repeat;
+    background-size: cover;
+    /* margin: 2rem; */
```
Comment (iRohitGaur) [inline index.css]: remove commented code

File: index.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    color: var(--TEXT-COLOR);
+    letter-spacing: 5px;
+    margin: 2rem;
+    padding: 2rem;
+}
+
+header p{
+    width: 50%;
+    margin: auto;
+    line-height: 20px;
```
Comment (iRohitGaur) [inline index.css]: use rem?

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes

File: index.css [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.flex-container .flex-card{
+    margin: 10px;
+}
+
+.card-body{
+    padding: 4rem 1rem;
+    text-align: left;
+}
```
Comment (abhishektanwar) [inline index.css]: Try to stay consistent with spacing between classes.

**Review by abhishektanwar — APPROVED** [review]
Clean code. Good to merge after changes suggested by @iRohitGaur

File: index.css (hunk 2 above)
Comment (purvasharmadev) [reply index.css]: Thank you for pointing it out. Fixed!

**Comment by purvasharmadev** [comment]
Thank You for reviewing and approving the code.

**Pull request merged**